source/develop/release-management/features/engine/metrics-store-installation-guide.html.md

```diff
@@ -17,6 +17,21 @@ feature_status: In Development
 **Note:** Currently, the oVirt Metrics Store should be installed on a new machine, separate from the engine.
 It can be installed on a dedicated VM.
 
+Before starting the installation please choose an environment name ("ovirt_env_name"). It will be used to identify data sent from more than one oVirt engine and collected in a single central store.
```

Comment:
ovirt_env_name variable should have a default and this default should be mentioned here